src/operation.ts

```diff
@@ -173,7 +173,7 @@ Operation.prototype.startPolling_ = function() {
     return;
   }
 
-  this.poll_(function(err, metadata) {
+  this.poll_((err, metadata) => {
     if (err) {
       self.emit('error', err);
```

Comment:
I'm not sure if this is something you'd want to tackle here as well, but you should be able to remove a lot of the `self` variables in favor of `this` if we're using arrow functions now.

Reply:
Absolutely!  That's something I was going to address in a follow up PR.  Wanted to keep this one nice and simple :)